bedrock/mozorg/views.py

```diff
@@ -125,7 +125,7 @@ def home_view(request):
         "donate_locale": donate_locale,
         "donate_params": donate_params,
         "pocket_articles": PocketArticle.objects.all()[:4],
-        "ftl_files": ["mozorg/home", "mozorg/home-mr2-promo"],
+        "ftl_files": ["mozorg/home", "mozorg/home-mr2-promo", "mozorg/home-fx100-promo"],
```

Comment:
No longer needed if there is no FTL file.
```suggestion
        "ftl_files": ["mozorg/home", "mozorg/home-mr2-promo"],
```